tests/go/GoTOMLBearTest.py

```diff
@@ -0,0 +1,34 @@
+from bears.go.GoTOMLBear import GoTOMLBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """package main
+import "fmt"
+func main() {
+    fmt.Println("Hello World!")
+}
+"""
+
+
+bad_file = """
```

Comment:
i think you should start the code from this line, not from next one, not 100% sure tho